### PR TITLE
Remove docker login from Jenkin script

### PR DESCRIPTION
**Pull request by anpanigr**
Started  Kind Nightly 
    https://build.weblogick8s.org:8443/job/wko-kind-nightly-seqential/974/
    https://build.weblogick8s.org:8443/job/wko-kind-nightly-parallel/1226/
https://build.weblogick8s.org:8443/job/wko-oke-nightly-seqential/287/

### CODE REVIEW

File: okdtest.sh [hunk 1]
```diff
@@ -150,7 +150,7 @@ ${KUBERNETES_CLI} get clusterrolebindings --no-headers | awk '/traefik-/{print $
 
 sudo rm -rf ${PV_ROOT}/*
 
-repoLogin
+#repoLogin
```
Comment (robertpatrick) [inline okdtest.sh:153]: Remove code, do not comment it out.  Git provides the history needed to recover if we ever need to do so.

File: Jenkinsfile.oke [hunk 2]
```diff
@@ -281,7 +281,7 @@ EOF
                         DOCKERHUB_PASSWORD = credentials("${dockerhub_password_creds}")
                     }
                     steps {
-                        sh 'echo ${DOCKERHUB_PASSWORD} | docker login -u ${DOCKERHUB_USERNAME} --password-stdin'
+                        # sh 'echo ${DOCKERHUB_PASSWORD} | docker login -u ${DOCKERHUB_USERNAME} --password-stdin'
```
Comment (robertpatrick) [inline Jenkinsfile.oke:284]: Remove code, do not comment it out.  Git provides the history needed to recover if we ever need to do so.

File: jenkinsScript.sh [hunk 3]
```diff
@@ -125,7 +125,7 @@ ulimit -a
 echo "Info: hard limits"
 ulimit -aH
 
-repoLogin
+#repoLogin
```
Comment (robertpatrick) [inline jenkinsScript.sh:128]: Remove code, do not comment it out.

**Review by robertpatrick — CHANGES_REQUESTED** [review]

File: okdtest.sh (hunk 1 above)
Comment (anpanigr) [reply okdtest.sh]: done

File: jenkinsScript.sh (hunk 3 above)
Comment (anpanigr) [reply jenkinsScript.sh]: done

**Review by robertpatrick — APPROVED** [review]

File: Jenkinsfile.oke [hunk 4]
```diff
@@ -281,7 +281,6 @@ EOF
                         DOCKERHUB_PASSWORD = credentials("${dockerhub_password_creds}")
```
Comment (rjeberhard) [inline Jenkinsfile.oke:281]: Please remove this environment section and the reference to these credentials earlier (line 20)

Comment (anpanigr) [reply Jenkinsfile.oke]: done

**Review by rjeberhard — APPROVED** [review]

**Pull request merged**